app/Libraries/Parser/GithubParser.php

```diff
@@ -204,35 +224,35 @@ private function buildSlackAttachmentFromCR()
         $authorLink = $this->request['pull_request']['user']['html_url'];
         $title = $this->request['pull_request']['title'];
         $titleLink = $this->request['pull_request']['html_url'];
-        $pretext = ':hammer_and_wrench: We wants you to make some changes to this Pull Request.';
+        $pretext = ':hammer_and_wrench: We want you to make some changes to this Pull Request.';
         $text = ':crossed_swords: Make the changes and update the Pull Request.';
 
-        $this->attachment->setColor('warning')
-                         ->setIconUrl(env('APP_URL').'/img/dashi-warning.png')
-                         ->setPretext($pretext)
-                         ->setAuthorName($authorName)
-                         ->setAuthorLink($authorLink)
-                         ->setAuthorIcon($authorIcon)
-                         ->setTitle($title)
-                         ->setTitleLink($titleLink)
-                         ->setText($text)
-                         ->setFields([
-                            'title' => 'Comment',
-                            'value' => $this->request['review']['body'],
-                            'short' => false,
-                         ])
-                         ->setFields([
-                            'title' => 'Repository',
-                            'value' => $this->request['repository']['name'],
-                            'short' => true,
-                         ])
-                         ->setFields([
-                            'title' => 'From',
-                            'value' => 'Github',
-                            'short' => true,
-                         ])
-                         ->setFooter('Dashi')
-                         ->setFooterIcon(env('APP_URL').'/img/dashi-logo.png');
+        $this->attachment->setColor('#ce0502')
```

Comment:
@gabbanaesteban we need to create a class `Color` to avoid these magic numbers and use it like this...
```php
Color::BLUE
```

Reply:
Mm what about "Color::INFO"? In order to follow with bootstrap standard colors.

Reply:
You're right @gabbanaesteban